src/pages/ProfilesExplorerView/components/SceneExploreServiceLabels/components/SceneGroupByLabels/components/SceneLabelValuesGrid/domain/types.ts

```diff
@@ -0,0 +1,4 @@
+export enum CompareTarget {
+  BASELINE = 'baseline',
+  COMPARISON = 'comparison',
```

Comment:
```suggestion
  BASELINE = 'baseline',
  COMPARISON = 'comparison',
```

The behaviour of only have either or is quite limiting. Assuming you have 3 pods a,b,c

I think it is very valid and a good usecase to compare:

baseline: b,c (or everything not a)
comparsion: a

Even this makes a lot of sense

baseline: b,c (or everything not a)
comparsion: a

Wonder if that is something you were planning to address later (once you have the "in" support)

Reply:
Indeed, that's a great idea but I'm not sure we should include it in this view, which is already quite complex (and might become difficult to understand for novice users).

My plan is to start prototyping a new comparison view, based on the use case that you've just shared and others.